samples/ChangeFeedProcessor/DocumentDB.ChangeFeedProcessor/ChangeFeedEventHost.cs

```diff
@@ -697,6 +703,9 @@ static DocumentCollectionInfo CanoninicalizeCollectionInfo(DocumentCollectionInf
 
         async Task StartAsync()
         {
+            // TODO: move this to constructor, but 1st check with functions intergration which seems to use it internally for estimating work.
+            if (string.IsNullOrWhiteSpace(this.HostName)) throw new ArgumentException("The hostName parameter provided to the constructor cannot be null or empty string.", "hostName");
```

Comment:
@mkolt Will confirm once I get back from them

Reply:
@mkolt Confirmed, they use a fixed name in the constructor parameter.

Reply:
please sign-off :)